### PR TITLE
Migrate works phase#2

### PR DESCRIPTION
**Pull request by amontenegro**
https://trello.com/c/dYLhGhYs/2047-migrate-work-info-to-the-work-table-phase-2
https://trello.com/c/wo72sBWb/2048-migrate-work-info-to-the-work-table-phase-3
https://trello.com/c/DbJDuGHd/2079-memberv2test-testviewactivitiessummaries-is-failing

### CODE REVIEW

**Comment by amontenegro** [comment]
Please merge AFTER: https://github.com/ORCID/ORCID-Source/pull/2059

File: orcid-core/src/main/java/org/orcid/core/adapter/impl/Jpa2JaxbAdapterImpl.java [hunk 1]
```diff
@@ -308,21 +308,74 @@ private FundingList getFundingList(ProfileEntity profileEntity) {
         return null;
     }
 
+
```
Comment (rcpeters) [inline orcid-core/src/main/java/org/orcid/core/adapter/impl/Jpa2JaxbAdapterImpl.java]: lots of new lines?

Comment (amontenegro) [inline orcid-core/src/main/java/org/orcid/core/adapter/impl/Jpa2JaxbAdapterImpl.java]: HAHA sorry! maybe i fell asleep in the keyboard :P

**Comment by rcpeters** [comment]
merge monster.

**Comment by amontenegro** [comment]
@rcpeters please dont merge yet! there are some failing tests that im working on, I will let you know when we are ready to go

**Pull request merged**